app/scripts/services/aggregatedLoggingService.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+angular.module("openshiftConsole")
+  .factory("AggregatedLoggingService", function($q, Logger, DataService) {
+
+    // cache previous request
+    var userAllowed;
+
+    // Create SelfSubjectAccessReview request againt authorization API to check whether
+    // current user can view pods/log from 'default' project.
+    // Users who can view such logs are 'operations' users
+    var isOperationsUser = function() {
+      var authPromise = null;
+
+      if (userAllowed === undefined) {
+        Logger.log("AggregatedLoggingService, loading whether user is Operations user");
+        var ssar = {
+          apiVersion: 'authorization.k8s.io/v1',
+          kind: 'SelfSubjectAccessReview',
+          spec: {
+            resourceAttributes: {
+              resource: 'pods/log',
+              namespace: 'default',
+              verb: 'view'
+            }
+          }
+        };
+        authPromise = DataService.create({ group: 'authorization.k8s.io', version: 'v1', resource: 'selfsubjectaccessreviews'},
```

Comment:
You don't need to create a Promise yourself. Just return the Promise from DataService.create

```js
return DataService.create(...).then(...);
```